changelog.md

```diff
@@ -108,18 +99,221 @@ deliver information you explicitly requested by default, even though we will
 allow some compatibility configuration option. This improves performance by not
 returning information that you might not need.
 
+#### Smarter resource offer handling
+
+When Marathon needed to launch tasks for multiple apps, it would first launch all needed tasks for the first one
+and then proceeding with the next app. One big deployment could block all others. In this new version, Marathon
+will spread the offers among all apps that currently need to launch tasks.
+
+When Marathon detects new tasks that it has to launch, then it will now explicitly request new offers
+with the `reviveOffers` Mesos scheduler API call. This should result in more speedy task launches. If
+for any reason, you dislike this behavior, you can disable it with `--disable_revive_offers_for_new_apps`
+(not recommended).
+
+When Marathon has no current use for an offer, it will decline the offer for a configurable period. This period is
+configurable. A short duration might lead to resource starvation for other frameworks if you run many frameworks
```

Comment:
`...for a configurable period. This period is configurable`
This duplication can be removed.